### ISSUE TITLE
Tags on Examples not working as expected

### ISSUE THREAD

**Issue by sayadzad** [jonsamwell/flutter_gherkin#98]
Hi, when I try to add tags to `Examples` blocks such as:

```
@tag1
Examples:
  | text |
  | abc |

@tag2
Examples:
  | text |
  | def |
```

I get an exception:
```
Instance of 'GherkinSyntaxException'
#0      ScenarioOutlineRunnable.expandOutlinesIntoScenarios (package:gherkin/src/gherkin/runnables/scenario_outline.dart:39:7)
#1      FeatureRunnable.addChild (package:gherkin/src/gherkin/runnables/feature.dart:40:34)
#2      GherkinParser._parseBlock (package:gherkin/src/gherkin/parser.dart:120:21)
#3      GherkinParser._parseBlock (package:gherkin/src/gherkin/parser.dart:109:15)
#4      GherkinParser.parseFeatureFile (package:gherkin/src/gherkin/parser.dart:55:7)
<asynchronous suspension>
#5      GherkinRunner.execute (package:gherkin/src/test_runner.dart:47:43)
<asynchronous suspension>
```

I believe this would be useful to support.

**Comment by jonsamwell** [comment]
@sayadzad if the `Example` is within a `Scenario Outline` I don't think they can have tags on them?  Instead you would put it on the parent `Scenario Outline`

**Comment by jonsamwell** [comment]
OK it looks like you can have multiple example blocks under a scenario outline so that now makes sense that examples should adhere to tag expressions https://relishapp.com/cucumber/cucumber/docs/gherkin/scenario-outlines

**Comment by jonsamwell** [comment]
Fixed in latest v1.1.9 release
